src/Raven.Studio/typescript/viewmodels/resources/deleteResourceConfirm.ts

```diff
@@ -10,10 +10,12 @@ class deleteResourceConfirm extends confirmViewModelBase<deleteResourceConfirmRe
 
     keepFiles() {
         this.isKeepingFiles(true);
+        super.confirm();
```

Comment:
please use this.confirm instead of super.confirm

Does we really need double confirmation?